pydm/display.py

```diff
@@ -82,6 +108,31 @@ def _load_ui_into_display(uifile, display):
     display.ui = display
 
 
+def _load_compiled_ui_into_display(code_string: str, class_name: str, display) -> None:
```

Comment:
You can also type hint `display: Display` here if you want, but it requires the `from __future__ import annotations` import which postpones the evaluation of type annotations (giving you time to establish the `Display` class below)

Reply:
Ah, thanks! I will add that